### ISSUE TITLE
Malformed table in the docs

### ISSUE THREAD

**Issue by Andrew-S-Rosen** [cclib/cclib#1399]
https://cclib.github.io/data.html#details-of-current-implementation

The table here got all screwy :(

**Comment by xymaxim** [comment]
This is related with #1392. The table is rendered correctly locally with the code from master for me.

**Comment by Andrew-S-Rosen** [comment]
Thanks. Closing as duplicate.

**Comment by berquist** [comment]
> This is related with #1392. The table is rendered correctly locally with the code from master for me.

I don't think so.  The issue linked has to do with the GHA workflow trying to build clean docs.  I haven't checked but I bet it works fine with a dirty working copy.  Things still aren't isolated properly.

This is the same problem as #1154, except this time I reintroduced the problem when messing with the pyquante2 build system, and the stdout message about missing Cython Becke routine is coming from pyquante2.  I've been meaning to fix it properly but 2.0 was occupying my brain.

Though, in order to push generated docs properly, #1392 does need to be fixed...

**Comment by berquist** [comment]
Caused by https://github.com/rpmuller/pyquante2/pull/10

**Comment by berquist** [comment]
Waiting on https://github.com/rpmuller/pyquante2/pull/11
